rabbitmq/tests/metrics.py

```diff
@@ -26,8 +26,8 @@
     'rabbitmq.exchange.messages.publish_out.rate',
 ]
 
-# Only present in 3.5
-if RABBITMQ_VERSION == version.parse('3.5'):
+# Only present in 3.5 or greater
+if RABBITMQ_VERSION >= version.parse('3.5'):
```

Comment:
Tests are failing because those metrics are not collected for 3.6 at least. You sure about that?

Reply:
You're right. my mistake